src/components/compounds/Interests/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Interests = ({ interests }) => (
+  <div className="ProfileInterest">
+    <div className="ProfileInterest__title">
+    Interests
+    </div>
+    <div className="ProfileInterest__tags">
+      {
+        (interests)
```

Comment:
this doesn't need to be in parenthesis.

Reply:
Also, it is usually better if we have array computation in methods. Just a style guide doesn't really affect performance.